### ISSUE TITLE
Feature request: [Strato] set custom MX Record

### ISSUE THREAD

**Issue by wawerzineks** [qdm12/ddns-updater#162]
1. What's the feature?
I am using the MX Record option for strato, e.g.:
	https://dyndns.strato.com/nic/update?hostname=domain.com&myip=ip&system=dyndns&wildcard=NOCHG&mx=mail.provider.com&backmx=NOCHG"`
 Would it be possible to integrate such a option?

**Comment by fredericrous** [comment]
Hey @wawerzineks, what is the wildcard option? Documentation is hard to find for strato, I can see the options `mx` and `backmx` are valid and that the default value is NOCHG like No Change, but I don't see `wildcard`. The best doc I found is here (in German): https://www.strato.de/faq/domains/so-einfach-richten-sie-dyndns-fuer-ihre-domains-ein/

**Comment by qdm12** [comment]
I think @wawerzineks wants support for MX records on top of A and AAAA records. I guess it's possible to add it for strato only for now.

Regarding user inputs in the json config, would a `"custom_record_type"` field be ok? Or maybe have a `"record_type"` for all providers where you could set it to `A`, `AAAA` or `MX` (for strato only for now)?

**Comment by wawerzineks** [comment]
@fredericrous good question, I think it is an obsolete option and was related to something like subdomains. But for now this topic is not covered by the official Strato documentation anymore. All options you can set in the Strato Admin Center are the following: 
![grafik](https://user-images.githubusercontent.com/20044064/107206123-edf81400-69fe-11eb-930d-ce3347b7d864.png)

@qdm12 right, I think the first option would be sufficient, otherwise existing configs would have to be changed right?

**Comment by qdm12** [comment]
> I think the first option would be sufficient, otherwise existing configs would have to be changed right?

We could change the scheme and still support older fields, but yeah it's not ideal for the poor developers haha

So just to be clear, you would want to set your IPv4 (or IPv6) address as a value for your MX-record right? I think adding a custom_record_type would be nice for now, we can always do a v2 one day to break things eventually.

**Comment by wawerzineks** [comment]
Yep, but not my IPv4, more an DNS Name of an Mail-Provider, like aspmx.l.google.com

**Comment by qdm12** [comment]
Sorry if I'm missing the point but why would you need this program to add a constant value to your domain?

**Comment by wawerzineks** [comment]
When I activate the DynDNS option at Stratos Admin Center the MX Record can exclusively be set by the dyndns client and the button for managing the mx record gets deactivated. I guess this is just a behavior of Strato :(

**Comment by qdm12** [comment]
Interesting, although maybe a bit out of scope. Maybe reach out to them so they fix their UI or give a reason for that 'graying'? If they're not willing to fix it, we could have a field specific to strato for mx records so that it sets it at the program start only once?
